clang/lib/Basic/Targets/RISCV.h

```diff
@@ -28,6 +28,7 @@ class RISCVTargetInfo : public TargetInfo {
 protected:
   std::string ABI, CPU;
   std::unique_ptr<llvm::RISCVISAInfo> ISAInfo;
+  bool HasExperimental = false;
```

Comment:
Can this be private like `FastUnalignedAccess`?

Reply:
Done